airflow/providers/google/cloud/operators/cloud_build.py

```diff
@@ -83,13 +85,29 @@ def _reformat_storage_source(self):
 
         self.body["source"]["storageSource"] = self._convert_storage_url_to_dict(source)
 
+    def _load_body_to_dict(self):
+        """
+        :param file:
+            file path to YAML build config
```

Comment:
Can you also add JSON file support? This is not common, but build can also be defined as JSON files.

> You can write the build config file using the YAML or the JSON syntax.

https://cloud.google.com/cloud-build/docs/build-config#structure_of_a_build_config_file